Doc/library/stdtypes.rst

```diff
@@ -4372,9 +4372,9 @@ Dictionaries can be created by placing a comma-separated list of ``key: value``
 pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
 'jack', 4127: 'sjoerd'}``, or by the :class:`dict` constructor.
 
-.. class:: dict(**kwarg)
-           dict(mapping, **kwarg)
-           dict(iterable, **kwarg)
+.. class:: dict(**kwargs)
+           dict(mapping, **kwargs)
+           dict(iterable, **kwargs)
```

Comment:
Why change these too? The entire docs has inconsistent use between ``kwargs``, ``kwds``, ``kws`` etc., at this point it would cause too much docs churn if we went through all of them.

Reply:
I was not aware that entire docs have inconsistent use of `kwargs`. Please ignore this change.

Reply:
I'm ok with both variants but `kwargs` is used more often IMHO